apps/web/cypress/support/commands.ts

```diff
@@ -17,6 +17,15 @@ Cypress.Commands.add('seed', () => {
   return cy.request('POST', `${Cypress.env('apiUrl')}/v1/testing/seed`, {});
 });
 
+Cypress.Commands.add('eventsTrigger', (body: Cypress.RequestBody, apiKey: string) => {
+  return cy.request({
+    method: 'POST',
+    url: `${Cypress.env('apiUrl')}/v1/events/trigger`,
+    body,
+    headers: { 'Content-Type': 'application/json', Authorization: `ApiKey ${apiKey}` },
+  });
+});
```

Comment:
Added this command that allows triggering the event. I do use it in the test below.

Reply:
This is a great idea for reuse! We already have a `createNotification` task in cypress that does something similar a use of it can be found in the activity page tests:
```
cy.initializeSession()
      .as('session')
      .then((session: any) => {
        cy.wait(500);

        return cy.task('createNotifications', {
          identifier: session.templates[0].triggers[0].identifier,
          token: session.token,
          count: 25,
          organizationId: session.organization._id,
          environmentId: session.environment._id,
        });
      });
```

Do you think we need another one as command here?

Reply:
ohh cool! I was looking for that by didn't found, I'll try to reuse that ;)